edb/pgsql/compiler/clauses.py

```diff
@@ -400,3 +401,13 @@ def populate_argmap(
             index=index,
             required=param.required,
         )
+    for param in globals:
+        ctx.argmap[param.name] = pgast.Param(
+            index=len(ctx.argmap) + 1,
+            required=param.required,
+        )
+        if param.has_present_arg:
+            ctx.argmap[param.name + "_present"] = pgast.Param(
```

Comment:
better use `f'__{param_name}_present__'` -- we ban `__` names specifically to be able to avoid collisions in cases like this.